notebooks/notebooks/iclr2022/win-rate-vs-board-size.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    fig, _ = plot_adv_win_rate_by_board_size(df, r"$\texttt{Latest}$")
</pre><p>Should this be Latest or Latest_def?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
victim is unhardened in these games, so Latest is correct